lib/debugger/breakpoints.js

```diff
@@ -243,9 +245,47 @@ export default class BreakpointManager {
     this.emitter.emit('update')
   }
 
+  toggleConditionAtSourceLocation ({file, line}) {
+    const activePane = atom.workspace.getActivePane()
+
+    let found = false
+    this.fileBreakpoints.forEach(bp => {
+      if (bp.file === file && bp.line === line) {
```

Comment:
IMHO 
``` 
if (bp.file !== file && bp.line !== line) return
```
reads a bit better.

Reply:
Maybe you meant `||` instead of `&&` ?
I modified with `if (bp.file !== file || bp.line !== line) return`

Reply:
Ah yeah, of course.